src/api/public/api.ts

```diff
@@ -13,4 +13,12 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+import BpmnVisualization from '../../component/BpmnVisualization';
+import { GlobalOptions, NavigationConfiguration, FitOptions, FitType, LoadOptions, ZoomConfiguration } from '../../component/options';
+
+// Initialization and configuration options
+export { BpmnVisualization };
+export { GlobalOptions, NavigationConfiguration, FitOptions, FitType, LoadOptions, ZoomConfiguration };
```

Comment:
To have less lines, why do not do something like it does in the line 24:
```suggestion
// Initialization and configuration options
export { BpmnVisualization } from '../../component/BpmnVisualization';
export { GlobalOptions, NavigationConfiguration, FitOptions, FitType, LoadOptions, ZoomConfiguration }from '../../component/options';
```

Reply:
Already managed with 2000645